recipes/cccl-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cccl-python" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```

Comment:
Should this package be versioned the same way as the PyPI package, which is more like `0.1.3.2.0`? https://pypi.org/project/cuda-cccl/

Reply:
The Python library will be versioned separately from the C++ library is my current understanding.